### PR TITLE
USB Remote: Add new buttons

### PR DESCRIPTION
**Pull request by xMasterX**
# What's new

- Added two new buttons -> Esc & Del

# Verification 

- Launch app, scroll down, see new buttons, click to verify that they represent correct keyboard buttons

# Checklist (For Reviewer)

- [ ] PR has description of feature/bug or link to Confluence/Jira task
- [ ] Description contains actions to verify feature/bugfix
- [ ] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/plugins/hid_app/views/hid_keyboard.c [hunk 1]
```diff
@@ -140,18 +140,18 @@ const HidKeyboardKey hid_keyboard_keyset[ROW_COUNT][COLUMN_COUNT] = {
         {.width = 1, .icon = &I_ButtonRight_4x7, .value = HID_KEYBOARD_RIGHT_ARROW},
     },
     {
-        {.width = 3, .icon = NULL, .key = "Ctrl", .value = HID_KEYBOARD_L_CTRL},
+        {.width = 2, .icon = NULL, .key = "Ctl", .value = HID_KEYBOARD_L_CTRL},
```
Comment (skotopes) [inline applications/plugins/hid_app/views/hid_keyboard.c:143]: Not enough space to draw them properly. I'll ask our UI/UX team to make proper icons.

**Review by skotopes — CHANGES_REQUESTED** [review]

File: applications/plugins/hid_app/views/hid_keyboard.c (hunk 1 above)
Comment (xMasterX) [reply applications/plugins/hid_app/views/hid_keyboard.c:143]: > Not enough space to draw them properly. I'll ask our UI/UX team to make proper icons.

Okay, Ill wait for icons and will make changes to this PR after that

**Comment by skotopes** [comment]
Waiting for @Karator to provide new icons

**Comment by skotopes** [comment]
Implemented in https://github.com/flipperdevices/flipperzero-good-faps/pull/27

**Pull request closed without merging**